### DISCUSSION TITLE
Show screens that are not part of a `LiquidMenu` object

### DISCUSSION THREAD

**Discussion by Fettkeewl** [VasilKalchev/LiquidMenu#61] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
Screens such as
* welcome screen
* Confirmation dialogs
* "Standby" screens

Do not seem to be possible without including them in the LiquidMenu object?
Or am I supposed to have 2 LiquidMenu options to achieve this?

I have some screens that are displayed using a rotary encoder and wish to have some screens
that are displayed at specific events. With limited RAM on my nano I wish not to create a new LiquidMenu object just for this.

**Describe the solution you'd like**
Allow 
bool LiquidMenu::change_screen	(	LiquidScreen & 	p_liquidScreen	)	

to take and show screens even if they are not in the menu object!
I cant get it to work right now.

I want to use a welcome screen not part of the standard menu aswell as a timeout screen when no user input has been made for some while. Theese should not be a part of the user interactable menu,

**Describe alternatives you've considered**
2x LiquidMenu objects.. requires to much code

**Comment by VasilKalchev** [comment, marked as answer]
You can achieve this by using the method `void LiquidScreen::hide(bool hide)`, it will skip the "screen" when cycling the "screens". Refer to the documentation _(README.md -> Resources -> API reference)_ for more information.
